app/controllers/posts_controller.rb

```diff
@@ -1,6 +1,6 @@
 class PostsController < ApplicationController
   def index
-    @user = User.find(params[:authorId])
+    @user = User.find_by(params[:authorId])
     @posts = @user.posts
```

Comment:
- [ ] It would be better if we solve the `n+1` query problem when fetching all posts and their comments for a user :man_technologist: